ffi/mupdf.lua

```diff
@@ -362,7 +362,7 @@ calculate page size after applying DrawContext
 --]]
 function page_mt.__index:getSize(draw_context)
     local bounds = ffi.new("fz_rect[1]")
-    local bbox = ffi.new("fz_irect[1]")
+    -- local bbox = ffi.new("fz_irect[1]")
```

Comment:
Did @NiLuJe overlook this?

Reply:
Yes (but you too did not look at CI output with #1404 :)

Reply:
It's because that stupid Mac build is training me to ignore failed checks. :-(

(And why did it not fail now? Ugh.)

Reply:
It did fail for my first commit - that's why I added the 2nd to fix the failure.

Reply:
Oh, right, I must have did my final test run while still running the "with debug print" version that still needed it to print it, my bad ;).